lib/ansible/modules/cloud/amazon/elb_application_lb.py

```diff
@@ -434,9 +444,36 @@ def get_elb_attributes(connection, module, elb_arn):
     return elb_attributes
 
 
+def get_listener(connection, module, elb_arn, listener_port):
+    """
+    Get a listener based on the port provided.
+
+    :param connection:
+    :param module:
+    :param listener_port:
+    :return:
+    """
+
+    try:
```

Comment:
you could just use `get_elb_listeners` for this section of code, it seems to be identical.